tests/dummy/app/templates/docs/motions.md

```diff
@@ -0,0 +1,61 @@
+# Motions
+
+A motion animates a single sprite. Each motion takes a sprite as the first argument and options as the second argument. Motions can also have `to` and `from` arguments. These arguments take initial and final attribute values of the sprite. Motions animate the change of the given attribute from its initial to its final value. Like transitions, you may use any of the built in motions or create your own. 
+
+
+### move
+Animates a sprite from its current position to its final position using css transforms. The only option argument it takes is `easing`. Because this motion works with `keptSprites`, it requires the start point and the end point of a sprite. 
+
+{{#docs-demo as |demo|}}
+    {{#demo.example name="move"}}
+      {{transitions-example}}
+    {{/demo.example}}
+
+    {{demo.snippet 'transitions-snippet.hbs' label='transitions-example.hbs'}}
+    {{demo.snippet 'transitions-snippet.js' label='transitions-example.js'}}
+    {{demo.snippet 'transitions-snippet.css'}}
+{{/docs-demo}}
+
+### moveSVG
+Animates one property on a svg element. This motion takes in an attribute name and animates that attribute from its initial to its final value. For example, on a circle `moveSVG` animates the `cx` property.
```

Comment:
This should maybe have a lead-in, "ember-animated works for SVGs too! An SVG is an image format made of xml. Many vector illustration programs can export SVGs." New devs who want to make fun animations may have no clue what an SVG is. (P.S. I love SVGs)

Reply:
Also the "Move" should probably be a bold, interactive button at the top of the demo. I tried clicking on circles and nothing happened.